### PR TITLE
Migrate DropdownButton, NavItem, ListGroup to Material UI Issue #173

### PR DESCRIPTION
**Pull request by annovo**
Pull-Request for `paretOS`

## Description
- Changed MenuItem & DropdownButton to material UI components 
- RouteNavItem deleted
- Changed ListGroup to div

## Relates to
- [Refactor: Migrate one-off Bootstrap Components Table, DropdownButton, NavItem, ListGroup to Material UI #173 ](https://github.com/mikhael28/paretOS/issues/173)

## Reviewers
- @jayeclark 

### Screenshots / other info

Screenshot

### CODE REVIEW

**Comment by mikhael28** [comment]
Hey @annovo - good evening! I'm going to pull down your code and give it a review! It looks good from my initial view. I see you are based in Washington, I'm based in Seattle myself - I'm going to the April Seattle JS Meetup on Wednesday hosted by Carter Rabasa, would be great to meet in person! https://ti.to/event-loop/seattlejs-april-2022/

File: src/learn/ExperienceModule.js [hunk 1]
```diff
@@ -403,7 +402,7 @@ class ExperienceModule extends Component {
           />
         </h1>
         <div className="experience-container flex">
-          <ListGroup style={{ flexBasis: "30%" }} className="overflow">
+          <div style={{ flexBasis: "30%" }} className="overflow">
```
Comment (mikhael28) [inline src/learn/ExperienceModule.js:405]: Thank you for getting rid of another `react-bootstrap` dependency!

**Comment by mikhael28** [comment]
@annovo did you put this together yourself? https://github.com/annovo/gameoflife

I've read about the Game of Life, I think it would be cool to include a fully featured version of the game in the ParetOS - how 'complete' is this implementation you put together? It could be 100% for all I know :)

File: src/learn/ExperienceModule.js (hunk 1 above)
Comment (annovo) [reply src/learn/ExperienceModule.js:405]: Thank you, but it was @jayeclark who noticed it first and I was just following the lead 🙂

**Comment by annovo** [comment]
> Hey @annovo - good evening! I'm going to pull down your code and give it a review! It looks good from my initial view. I see you are based in Washington, I'm based in Seattle myself - I'm going to the April Seattle JS Meetup on Wednesday hosted by Carter Rabasa, would be great to meet in person! https://ti.to/event-loop/seattlejs-april-2022/

Hi! Wow, what a coincidence, I will definitely try to make it 👍 

It is my first contribution to an open source project so I'm really glad that everything is okay!

> how 'complete' is this implementation you put together?

Yes, I did it some time ago and I think it's pretty much complete. I mean there are a few suggestions that could be implemented (i.e. add some initial examples/patterns), but besides that, it's working and ready to be used. You can check out this [link](https://annovo.github.io/gameoflife/) :relaxed:

**Comment by mikhael28** [comment]
I really like your implementation - I've been thinking about having a 'wallpaper' or 'screensaver' mode for the ParetOS, where we can have a full-screen visualization (with some background deep house music, or something like that) that people can enable. Perhaps we can implement your code towards something like this?

Having a music player in the OS is definitely a priority, and I would love to honor Reuben Klamer somehow - having an interactive background/study music visualizer with your Game of Life visuals could be a compelling feature. If that sounds vaguely interesting, make an Issue in the Github repository with a few screenshots of your current implementation and maybe a couple of suggestions for how to write a simple, efficient music player. There are some interesting APIs here: https://www.elementary.audio

**Comment by annovo** [comment]
I watched the links you send and it does sound exciting. I will definitely give it more thought, however, I'm not sure if I will have time to go deep into this

**Comment by mikhael28** [comment]
No rush! Only work on what gives you energy and life 🚀

**Pull request merged**